app/dragontts/app/providers/sarvam_pool.py

```
@@ -0,0 +1,328 @@
+                try:
+                    m = json.loads(raw)
+                except (ValueError, TypeError):
+                    continue  # ignore non-JSON / control frames
+                mtype = m.get("type")
+                if mtype == "audio":
+                    audio_b64 = (m.get("data") or {}).get("audio")
+                    if audio_b64:
+                        try:
+                            yield base64.b64decode(audio_b64)
+                        except Exception:
+                            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)